docker-app/qfieldcloud/core/models.py

```diff
@@ -1464,7 +1464,7 @@ def attachment_dirs(self) -> list[str]:
         Attachment dir is a special directory in the QField infrastructure that holds attachment files
         such as images, pdf etc. By default "DCIM" is considered a attachment directory.
 
-        TODO this function expects whether `attachment_dirs` key in project_details. However,
+        TODO this function expects whether `attachmentDirs` key in project_details. However,
```

Comment:
Why the change? `attachment_dirs` is the name of the key in the `Project.project_details` JSON? See https://github.com/opengisch/QFieldCloud/pull/1271/files#diff-c5c429cde51e37e7ee4197d0dcf8d50d463ed5b560a6945dbb37504bddf2a74fR118

Actually I believe the TODO here is outdated.

Reply:
@suricactus , should I remove the TODO for both attachments and data dirs?

Reply:
Please do, yes. And squsah all into a single commit.